### PR TITLE
fix(manager-api): add API Definitions to export-import

### LINKED ISSUE #1279: API definition data is missing from the export-import definition.

### PR DESCRIPTION
**Pull request by msavy**
API Definitions are missing from the export-import files. 

Fixes #1279

### CODE REVIEW

File: manager/api/export-import/src/main/java/io/apiman/manager/api/exportimport/manager/StorageExporter.java [hunk 1]
```diff
@@ -97,33 +103,37 @@ public void export() {
...
-          Iterator<OrganizationBean> iter = storage.getAllOrganizations();
+            Iterator<OrganizationBean> iter = storage.getAllOrganizations();
+            int foo = 0;
 
-          while (iter.hasNext()) {
-              OrganizationBean bean = iter.next();
-              writer.startOrg(bean);
-              logger.info(Messages.i18n.format("StorageExporter.ExportingOrgs") + bean); //$NON-NLS-1$
+            while (iter.hasNext()) {
+                System.out.println("Organisation " + foo++);
```
Comment (msavy) [inline manager/api/export-import/src/main/java/io/apiman/manager/api/exportimport/manager/StorageExporter.java:120]: Remove this println

**Review by msavy — COMMENTED** [review]
Self-review OK, couple of nits to fix (stray diag statement)

**Comment by volkflo** [comment]
Fixed https://github.com/apiman/apiman/issues/1082 as well :)

**Comment by msavy** [comment]
Thanks a bunch! Couldn't find it; and apparently I wrote it 🤣.


>

**Comment by msavy** [comment]
@volkflo @BastianGem I've changed the RestTest code a bit so that you need to provide the flag `X-RestTest-RegexMatching: true` to trigger regex matching on the response.

I realised this was causing some issues where it can unexpectedly trigger regex patterns (for example when brackets are used like this), which causes the tests to fail in a confusing way.

So now it's explicitly on (which makes it obvious that regex escaping is needed).

**Review by msavy — COMMENTED** [review]
Self-review OK

**Review by volkflo — APPROVED** [review]
These changes look good to me. 
Import / Export may have a base64 bug.

**Comment by volkflo** [comment]
Btw do you know if there is a default file upload limit on wildfly/tomcat?
The new export/import files are getting probably larger than before.

**Comment by msavy** [comment]
> Btw do you know if there is a default file upload limit on wildfly/tomcat?
> The new export/import files are getting probably larger than before.

Good question. I think it can be configured in standalone.xml under  max-post-size.

**Comment by msavy** [comment]
Looks like it's 10MB by default. Perhaps we should set it to something like 200MB?

**Review by msavy — COMMENTED** [review]
self-review looks OK now

**Review by volkflo — APPROVED** [review]
LGTM now :)

**Pull request merged**